portality/templates/_js_includes.html

```diff
@@ -17,18 +16,26 @@
       gtag('config', 'UA-46560124-1');
     </script>
 
-    <!-- Hotjar Tracking Code for https://doaj.org -->
-    <script>
-        (function(h,o,t,j,a,r){
-            h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
-            h._hjSettings={hjid:1039257,hjsv:6};
-            a=o.getElementsByTagName('head')[0];
-            r=o.createElement('script');r.async=1;
-            r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
-            a.appendChild(r);
-        })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
-    </script>
-{% endif %}
+    {# To check current page #}
+    {%- set url_list = request.url.split("/") -%}
+    {%- set endpoint = url_list[2:url_list|length-1] -%}
+    {%- set endpoint = endpoint[endpoint|length-1] -%}
+
+    {# Only include Hotjar in dashboards #}
+    {%- if endpoint == 'admin' or endpoint == 'editor' or endpoint == 'publisher' -%}
+        <!-- Hotjar Tracking Code for https://doaj.org -->
+        <script>
+            (function(h,o,t,j,a,r){
+                h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+                h._hjSettings={hjid:1039257,hjsv:6};
+                a=o.getElementsByTagName('head')[0];
+                r=o.createElement('script');r.async=1;
+                r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+                a.appendChild(r);
+            })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+        </script>
+    {%- endif -%}
+{%- endif -%}
```

Comment:
FYI: after trying it out with the first option (I had to add `hotjar=True` to 30+ `render_template` functions), I opted for the second one because I just needed to do an include three times & create a partial for the markup! Hope that’s okay. 

Also, using `{% block extra_js_bottom %}{% endblock extra_js_bottom %}` didn’t seem to work for some reason...

Reply:
Pushed now to https://github.com/DOAJ/doaj/pull/1851/commits/5088c2565d7aa939073c7c1c36a6277e78d6e945